assets/css/front.css

```diff
@@ -646,8 +646,18 @@ a:has(.member-photo) {
     width: 100%;
     aspect-ratio: 3/1;
     overflow: hidden;
+    position: relative;
+}
+.gallery-employee:after {
+    content: "";
+    background-color: #33333361;
+    width: 100%;
+    position: absolute;
+    height: 100%;
+    left: 0;
+    top: 0;
```

Comment:
that is the standard of css.
i think background color should be used with css variable

Reply:
your argument is convincing